ledger/ledger-api-common/src/main/scala/com/digitalasset/ledger/api/validation/CommandsValidator.scala

```diff
@@ -278,6 +278,10 @@ final class CommandsValidator(
 }
 
 object CommandsValidator {
+  def apply(ledgerId: LedgerId, explicitDisclosureUnsafeEnabled: Boolean) = new CommandsValidator(
```

Comment:
why in general this property is called `explicitDisclosureUnsafeEnabled` ? What `Unsafe` means here?

Reply:
Unsafe as in the feature is unsafe. I've used this particle to pop in the eye for devs, if everyone were to enable it/review code with it enabled.